packer/instance-scripts/init-quorum.sh

```diff
@@ -42,15 +44,17 @@ function generate_quorum_supervisor_config {
 
     if [ "$ROLE" == "maker" ]
     then
-        ARGS="$GLOBAL_ARGS --blockmakeraccount \"$ADDRESS\" --blockmakerpassword \"$PASSWORD\" --minblocktime $MIN_BLOCK_TIME --maxblocktime $MAX_BLOCK_TIME"
+        ARGS="$GLOBAL_ARGS --blockmakeraccount \"$ADDRESS\" --minblocktime $MIN_BLOCK_TIME --maxblocktime $MAX_BLOCK_TIME"
     elif [ "$ROLE" == "validator" ]
     then
-        ARGS="$GLOBAL_ARGS --voteaccount \"$ADDRESS\" --votepassword \"$PASSWORD\""
+        ARGS="$GLOBAL_ARGS --voteaccount \"$ADDRESS\""
     else # observer node
         echo "$PASSWORD" > $PW_FILE
```

Comment:
Can we not create this file? Pretty sure we don't use it now with your change and it's a security hole